src/applications/gi/sass/partials/_gi-vet-tec.scss

```diff
@@ -22,6 +22,12 @@
 
   $single-column-width: 641px;
 
+  @media only screen and (max-width: 767px) {
+    .vettec-logo-desktop {
+      display: none;
+    }
+  }
```

Comment:
Could this be accomplished with a [responsive prefix](https://design.va.gov/utilities/display#responsive-prefixes) instead?  Maybe instead of `vettec-logo-desktop` you could add `vads-u-display--none medium-screen:vads-u-display--block`

Reply:
Updated CSS to use vads-u-display--none medium-screen:vads-u-display--block